src/pages/index.md

```diff
@@ -5,125 +5,89 @@ description: This is the overview page of Cat Analytics
 #  - https://github.com/icaraps 
 ---
 
-<Hero slots="heading, text"></Hero>
+<Hero slots="heading, text" background="rgb(246, 16, 27)"></Hero>
```

Comment:
remove red colour

Reply:
or change it like here: https://developer.adobe.com/firefly-services/docs/guides/